packs/feats/class/kineticist/rouse-the-forests-fury.json

```
@@ -12,7 +12,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Terrifying trees attack your enemies. Three Large trees appear in unoccupied spaces within 500 feet, at least 15 feet from one another. Each tree has AC 40, Fortitude +33, Reflex +24, Will +30, and 200 HP. The trees can flank, but are unable to move. When the trees appear, each makes a Strike; they share a multiple attack penalty. The tree's melee Strike is a branch that has reach 10 feet; on a hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] by the tree ([[/act escape dc=40]]). The ranged Strike is a hurled fruit, gourd, seed, or stone with a range increment of 60 feet. Either Strike has a +30 attack modifier and deals @Damage[(4d10+9)[bludgeoning]] damage.</p>\n<p>The trees last until the end of your next turn, and you can Sustain the impulse. Each time you Sustain the impulse, you can have one tree make a Strike.</p>\n<hr />\n<p><strong>Level (20th)</strong> The attack modifier is +32 and the Strike damage is 5d10+9.</p>"
+            "value": "<p>Terrifying trees attack your enemies. Three Large trees appear in unoccupied spaces within 500 feet, at least 15 feet from one another. Each tree has AC 40, Fortitude +33, Reflex +24, Will +30, and 200 HP. The trees can flank, but are unable to move. When the trees appear, each makes a Strike; they share a multiple attack penalty. The tree's melee Strike is a branch that has reach 10 feet; on a hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] by the tree ([[/act escape dc=40]]). The ranged Strike is a hurled fruit, gourd, seed, or stone with a range increment of 60 feet. Either Strike has a +30 attack modifier and deals @Damage[(ternary(gte(@actor.level, 20), 5, 4)d10+9)[bludgeoning]] damage.</p>\n<p>The trees last until the end of your next turn, and you can Sustain the impulse. Each time you Sustain the impulse, you can have one tree make a Strike.</p><hr /><p><strong>Level (20th)</strong> The attack modifier is +32 and the Strike damage is 5d10+9.</p>"
```

```suggestion
            "value": "<p>Terrifying trees attack your enemies. Three Large trees appear in unoccupied spaces within 500 feet, at least 15 feet from one another. Each tree has AC 40, Fortitude +33, Reflex +24, Will +30, and 200 HP. The trees can flank, but are unable to move. When the trees appear, each makes a Strike; they share a multiple attack penalty. The tree's melee Strike is a branch that has reach 10 feet; on a hit, the target is @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] by the tree ([[/act escape dc=40]]). The ranged Strike is a hurled fruit, gourd, seed, or stone with a range increment of 60 feet. Either Strike has a [[/r 1d20+30]]{+30} attack modifier and deals @Damage[(ternary(gte(@actor.level, 20), 5, 4)d10+9)[bludgeoning]] damage.</p>\n<p>The trees last until the end of your next turn, and you can Sustain the impulse. Each time you Sustain the impulse, you can have one tree make a Strike.</p><hr /><p><strong>Level (20th)</strong> The attack modifier is [[/r 1d20+32]]{+32} and the Strike damage is 5d10+9.</p>"
```
We can still add these simple inline rolls for convenience, which is the standard for abilities like this.